test/controllers/CapitalGainsTaxControllerSpec.scala

```diff
@@ -102,11 +100,11 @@ class CapitalGainsTaxControllerSpec extends ControllerBaseSpec {
       document.getElementById("taxable-gains").text() mustBe "£20,000"
       document.getElementById("less-taxable-gains").text() mustBe "minus £10,600 -£10,600"
       document.getElementById("cg-pay-tax-on").text() mustBe "£9,400"
-      document.getElementById("tax-period").text() mustBe "2013 to 2014"
+      document.getElementById("tax-period").text() mustBe s"${taxYear - 1} to $taxYear"
```

Comment:
same here. We only test last year? For paye I guess it is enough as the calculator is the same for all the years.

Reply:
This one will test whatever year's in appConfig. I assumed it'll be the ideal way as both the current and the new year are tested when changed

Reply:
That will do. Now that it is generic it easy to change or test multiple years